server/server.go

```diff
@@ -20,6 +23,45 @@ type Config struct {
 
 // New creates a new server object with given configuration.
 func New(config Config) (microserver.Server, error) {
+	var err error
+
+	if config.MicroServerConfig.Logger == nil {
+		return nil, microerror.Maskf(invalidConfigError,
+			"config.MicroServerConfig or it's Logger must not be empty")
+	}
+
+	if config.Service == nil {
+		return nil, microerror.Maskf(invalidConfigError,
+			"config.Service must not be empty")
+	}
+
+	var middlewareCollection *middleware.Middleware
+	{
+		middlewareConfig := middleware.Config{
+			Logger:  config.MicroServerConfig.Logger,
+			Service: config.Service,
+		}
```

Comment:
We started to name those simply `c`, i.e. `c := middleware.Config`. This is what nested scopes are good at :)
Is there a reason you chose a longer name?

Reply:
No. I also prefer short variable names, but I was reading some other operator source and noticed that long variable names were pretty consistent pattern so I tried to stay consistent as well. I'll change this.